test/e2e/performanceprofile/functests/2_performance_update/updating_profile.go

```diff
@@ -637,7 +637,7 @@ var _ = Describe("[rfe_id:28761][performance] Updating parameters in performance
 			})
 		})
 		When("HighPower Consumption workload enabled", func() {
-			It("[test_id:50992][crit:high][vendor:cnf-qe@redhat.com][level:acceptance]should update kernel arguments and tuned accordingly", func() {
+			It("[test_id:50992][crit:high][vendor:cnf-qe@redhat.com][level:acceptance]should update kernel arguments and tuned accordingly", Label("stalld-disabled"), func() {
```

Comment:
I would wait with the labels to a different PR that will have all labels

Reply:
yes, sorry removing it